connectors/sync_job_runner.py

```diff
@@ -90,7 +92,9 @@ async def execute(self):
                 options=bulk_options,
             )
 
+            self._reporting_cycle_start_time = time.time()
             while not self.elastic_server.done():
```

Comment:
I'm not 100% sure how it'll work - I think previous call to elastic_server is blocking?

Reply:
PR #513 made changes to `async_bulk`, although it's `awaited` and blocking, it will return once it has scheduled (i.e. `create_task`) the fetcher task and bulker task.

`self.elastic_server.done()` will return `True` if both the fetcher task and bulker task are done.